scrapy/exporters.py

```diff
@@ -214,9 +226,18 @@ def _write_headers_and_set_fields_to_export(self, item):
                     # use fields declared in Item
                     self.fields_to_export = list(item.fields.keys())
             row = list(self._build_row(self.fields_to_export))
-            self.csv_writer.writerow(row)
-
-
+            self._writerow(row)
+            
+    def _writerow(self, row):
```

Comment:
Can you clarify why is this needed?
It looks unrelated to this PR, what am I missing?